service_configuration_lib/spark_config.py

```diff
@@ -36,6 +36,7 @@
 DEFAULT_EXECUTOR_INSTANCES = 2
 DEFAULT_EXECUTOR_MEMORY = '4g'
 DEFAULT_K8S_LABEL_LENGTH = 63
+DEFAULT_K8S_BATCH_SIZE = 512
```

Comment:
Maybe change this to 1024 or higher (Basically more consistent with clusterman config of max_weight_to_add)?
Also, add the link here for the max_weight_to_add config. They kind of tie together.
With 512 and cores as 1. We only utilize 512 of 2500 available capacity to scale.

Reply:
> add the link here for the max_weight_to_add config
it would be a private corp link in a public repo :/ 

> Maybe change this to 1024 or higher (Basically more consistent with clusterman config of max_weight_to_add)?
we can go back later and bump the value if needed, or if necessary certain batches can set up a higher value.

This type of config will help 90-95th percentile of batches in our system, if not more.

Reply:
Makes sense. gfi